client-v2/src/services/faciaApi.ts

```diff
@@ -19,6 +19,39 @@ import { CapiArticle } from 'types/Capi';
 import chunk from 'lodash/chunk';
 import { CAPISearchQueryResponse, checkIsResults } from './capiQuery';
 import flatMap from 'lodash/flatMap';
+import { EditionsIssue } from 'types/Edition';
+
+function fetchEditionsIssueAsConfig(editionId: string): Promise<FrontsConfig> {
+  return pandaFetch(`/editions-api/${editionId}`, {
```

Comment:
The endpoint for fetching editions is going to look like this: ` /editions-api/editions/:name/issues/:id` so can we pass the edition name for this method as well? We could also do it in a separate pr.

Reply:
I think I'll leave this for a second PR as I think we can probably tidy up our routes and have some builders and parsers all in one place 👍